easybuild/easyconfigs/i/ITSx/ITSx-1.1.2-GCCcore-9.3.0.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'Tarball'
+
+name = 'ITSx'
+version = '1.1.2'
+
+homepage = 'https://microbiology.se/software/itsx/'
+description = '''ITSx: Improved software detection and extraction of ITS1 and ITS2 from ribosomal ITS sequences of
+fungi and other eukaryotes for use in environmental sequencing.'''
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+
+source_urls = ['https://microbiology.se/sw/']
+sources = ['%(name)s_%(version)s.tar.gz']
+checksums = ['45a9e99583189488e6cdfc37172732a330901ed713d92c8d5e7631a39fc4e72e']
+
+dependencies = [('Perl', '5.30.2')]
+
+fix_perl_shebang_for = ['ITSx']
+
+sanity_check_paths = {
+    'files': ['ITSx'],
```

Comment:
We like to have sanity check commands like "`ITSx --help`" or so, would that work here?